### PR TITLE
Redirect play if able

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
The next step in moving off playframes is to automatically redirect built-in guides to the sidebar guides

preview @ http://format_duration.surge.sh

### CODE REVIEW

File: src/shared/services/commandInterpreterHelper.ts [hunk 1]
```diff
@@ -590,6 +591,28 @@ const availableCommands = [
     name: 'play',
     match: (cmd: any) => /^play(\s|$)/.test(cmd),
     exec(action: any, put: any, store: any) {
+      // Built in play guides where migrated to
+      // use the guide command instead
+      const legacyBuiltInGuides = [
+        'concepts',
+        'cypher',
+        'index',
```
Comment (jharris4) [inline src/shared/services/commandInterpreterHelper.ts:599]: should this be here?

File: src/shared/services/commandInterpreterHelper.ts [hunk 2]
```diff
@@ -590,6 +591,28 @@ const availableCommands = [
     name: 'play',
     match: (cmd: any) => /^play(\s|$)/.test(cmd),
     exec(action: any, put: any, store: any) {
+      // Built in play guides where migrated to
+      // use the guide command instead
+      const legacyBuiltInGuides = [
```
Comment (jharris4) [inline src/shared/services/commandInterpreterHelper.ts:596]: can we remove the play command source code now if they're never used anymore?

File: src/shared/services/commandInterpreterHelper.ts (hunk 1 above)
Comment (OskarDamkjaer) [reply src/shared/services/commandInterpreterHelper.ts:599]: Removing

**Review by jharris4 — APPROVED** [review]
LGTM

**Pull request merged**